tests/end2end/test_sanity_checks.py

```diff
@@ -0,0 +1,29 @@
+import warnings
+
+import pytest
+
+from qiboconnection.api import API
+
+
+@pytest.mark.order(0)
+def test_store_initial_job_and_runcard_count(api: API, request):
+    """Count number jobs and runcards before running the test suite"""
+    initial_job_count = len(api.list_jobs().dataframe)
+    initial_runcards_count = len(api.list_runcards())
+    request.config.cache.set("initial_job_count", initial_job_count)
+    request.config.cache.set("initial_runcards_count", initial_runcards_count)
+
+    assert isinstance(initial_job_count, int)
+
+
+@pytest.mark.order(-1)
```

Comment:
-1 means this test is executed the last one?

Reply:
Yes! Check here: https://pytest-order.readthedocs.io/en/stable/usage.html#order-using-ordinals